test_scripts/Policies/Policy_Table_Update/176_PTU_is_performed_if_HMI_sends_BC.OnSystemRequest_with_url_parameter.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- Issues: https://github.com/smartdevicelink/sdl_core/issues/3715,
+--         https://github.com/smartdevicelink/sdl_core/issues/3881
+---------------------------------------------------------------------------------------------------
+-- Description: SDL accept PTU from application in case HMI sends BC.OnSystemRequest with 'url' parameter
+-- Steps:
+-- 1. Core and HMI are started
+-- 2. App is registered
+-- 3. Core sends OnStatusUpdate UPDATING
+-- 4. HMI sends OnSystemRequest with 'url' parameter
+-- 5. After OnSystemRequest is sent to app, app 1 attempts SystemRequest PTU
+-- SDL does:
+--  - accept the PTU, responding success:true, resultCode:SUCCESS
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local utils = require('user_modules/utils')
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+runner.testSettings.restrictions.sdlBuildOptions = { { extendedPolicy = { "PROPRIETARY", "EXTERNAL_PROPRIETARY" } } }
+
+--[[ Local Functions ]]
+local function policyTableUpdate()
+  local ptuFileName = os.tmpname()
+  local requestId = common.hmi.getConnection():SendRequest("SDL.GetPolicyConfigurationData",
+      { policyType = "module_config", property = "endpoints" })
+  common.hmi.getConnection():ExpectResponse(requestId)
+  :Do(function()
+      common.hmi.getConnection():SendNotification("BasicCommunication.OnSystemRequest", {
+        requestType = "PROPRIETARY",
+        fileName = common.sdl.getPTSFilePath(),
+        url = "http://x.x.x.x:3000/api/1/policies/proprietary"
+      })
+      local ptuTable = common.getPTUFromPTS()
+      for i, _ in pairs(common.mobile.getApps()) do
```

Comment:
This function is specific for this script so the functionality for apps can be removed, also line 44